phpunit/blocks/render-block-navigation-test.php

```diff
@@ -21,7 +21,7 @@ public function test_block_core_navigation_get_post_ids_from_block() {
 		);
 		$parsed_block  = $parsed_blocks[0];
 		$context       = array();
-		$block         = new WP_Block( $parsed_block, $context, $this->registry );
+		$block         = new WP_Block( $parsed_block, $context );
```

Comment:
Interesting. It looks like the third argument [has to be an instance of WP_Block_Type_Registry ](https://github.com/ramonjd/wordpress-develop/blob/1d020c1012e6601863008eaa91838c32820106d7/src/wp-includes/class-wp-block.php#L117)

Maybe https://github.com/WordPress/gutenberg/pull/40752 was copying examples from [core tests](https://github.com/ramonjd/wordpress-develop/blob/c88553d07658ed9c2158892f5712196b53d5b193/tests/phpunit/tests/blocks/wpBlock.php#L57-L56) and not assigning `$this->registry` a value.

Regardless, maybe another approach would be to set it like the [Tests_Blocks_wpBlock test does](https://github.com/ramonjd/wordpress-develop/blob/c88553d07658ed9c2158892f5712196b53d5b193/tests/phpunit/tests/blocks/wpBlock.php#L29):

```php
	/**
	 * Set up each test method.
	 */
	public function set_up() {
		parent::set_up();

		$this->registry = new WP_Block_Type_Registry();
	}
```
Even so it doesn't seem to affect the tests results.

Reply:
basically what @talldan said 2 minutes ago 🤣

Reply:
I appreciate the extra pair of eyes, cheers! 😀